### PR TITLE
Add more encumbrance for Footwear

### PR DESCRIPTION
**Pull request by pisskop**
Adds encumbrance to most shoes/boots.

The aim being to increase the importance of foot encumbrance when kitting out.

-Most boots and 'hard footwear' saw increases.
-Discomfort should add to encumbrance.
-stuff that doesn't fit properly should be very uncomfortable to use.

### CODE REVIEW

File: data/json/items/armor.json [hunk 1]
```diff
@@ -356,7 +356,7 @@
         "volume" : 15,
         "warmth" : 20,
         "environmental_protection" : 2,
-        "encumbrance" : 20,
+        "encumbrance" : 30,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: Knee-high boots encumber the legs too

File: data/json/items/armor.json [hunk 2]
```diff
@@ -387,7 +387,7 @@
         "volume" : 17,
         "warmth" : 20,
         "environmental_protection" : 2,
-        "encumbrance" : 25,
+        "encumbrance" : 40,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: Thigh-high boots encumber the legs too

File: data/json/items/armor.json [hunk 3]
```diff
@@ -503,7 +503,7 @@
         "volume" : 8,
         "warmth" : 30,
         "environmental_protection" : 2,
-        "encumbrance" : 10,
+        "encumbrance" : 18,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: "Durable and comfortable" in the description. They are pretty thin and have low armor values. Compare them to regular boots.

File: data/json/items/armor.json [hunk 4]
```diff
@@ -3925,7 +3925,7 @@
         "volume" : 38,
         "warmth" : 5,
         "environmental_protection" : 6,
-        "encumbrance" : 25,
+        "encumbrance" : 28,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: Those encumber not just feet and legs but also torso.

**Comment by pisskop** [comment]
fair enough.

File: data/json/items/armor.json [hunk 5]
```diff
@@ -165,7 +165,7 @@
         "material" : ["cotton", "leather"],
         "volume": 6,
         "warmth": 20,
-        "encumbrance": 10,
+        "encumbrance": 15,
```
Comment (WizardOfOoo) [inline data/json/items/armor.json]: If you are going to make sneakers 4, you should leave this at ten.

**Comment by pisskop** [comment]
4 is essentially zero, and without any meaningful accessories or other clothing for the feet its a show of symbolism.

**Comment by pisskop** [comment]
refitting an item will reduce its enciumbrance by up to half.


cleats will go to 7/8 from 15.

With that encumbrance you can puton 2 pairs of socks and reach 10.


the large studs of cleats reduce traction on hard flat surfaces like pavement and eliminate your ability to 'feel' the terrain you are on.

to say they as trivial to run in as sneakers or bearfoot is untrue, and the cleats will interfere with percision movements, as they are 1-2 inch spikes of hard material you have to account for that, unlike solid rubber souls, doesnt absorb shock and does embed itself

**Comment by WizardOfOoo** [comment]
Good points.  I take my comments back.

**Comment by kevingranade** [comment]
It's tempting to say adding a traction stat would be nice, it would
differentiate cleats in particular (though they'd need an adjustment for
hard surfaces), but I'm not sure it'd be worth the trouble.

File: data/json/items/armor.json (hunk 5 above)
Comment (Coolthulhu) [inline data/json/items/armor.json]: There is no need to nerf already useless items. Cleats don't have any real use at the moment.

File: data/json/items/armor.json [hunk 6]
```diff
@@ -8445,7 +8445,7 @@
         "volume" : 12,
         "warmth" : 15,
         "environmental_protection" : 10,
-        "encumbrance" : 35,
+        "encumbrance" : 38,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: They're now more restrictive than heavy survivor boots, without any real advantage over them.

File: data/json/items/armor.json [hunk 7]
```diff
@@ -791,7 +791,7 @@
         "volume" : 17,
         "warmth" : 20,
         "environmental_protection" : 3,
-        "encumbrance" : 15,
+        "encumbrance" : 22,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: This makes them worse than leather armor boots, despite being harder to craft.

File: data/json/items/armor.json [hunk 8]
```diff
@@ -759,7 +759,7 @@
         "volume" : 17,
         "warmth" : 10,
         "environmental_protection" : 3,
-        "encumbrance" : 12,
+        "encumbrance" : 18,
```
Comment (Coolthulhu) [inline data/json/items/armor.json]: Chitinous boots can't be refitted

**Comment by WizardOfOoo** [comment]
Maybe cleats would be like skates, but only for off road.

If traction was added, we could put Hobnail Boots In.
Hobnail boots are extremely durable and give great traction on any off road conditions, anything but super flat surfaces like smooth concrete.  They typically had iron plates and steel toes.
https://en.wikipedia.org/wiki/Trench_boot

I know I would make or find some hobnail or cleats if I was a survivor.  I LARPed in steel cleats, They grip turf really really good.  Great for quick movements, DODGING, direction changes, also good for extra power when shield bashing.  ;p

They did turn wet concrete into a slip and slide though.  Tile and brick was ok, but smooth concrete was bad.

**Comment by pisskop** [comment]
Hobnailed shoes would be awesome

**Comment by WizardOfOoo** [comment]
It's a good idea, but now I'm thinking that the existing survivor boot fills that role.  What would make it unique?

**Comment by pisskop** [comment]
Its authentic

**Comment by WizardOfOoo** [comment]
Those are all excellent points.  I think this need it's own pull request.

**Pull request merged**